library/ceph_bcache.py

```diff
@@ -35,10 +35,12 @@ def main():
         argument_spec=dict(
             disks=dict(type='list',required=True),
             ssd_device=dict(required=True),
+            journal_guid=dict(),
         ),
     )
     disks = module.params.get('disks')
     ssd_device = module.params.get('ssd_device')
+    journal_guid = module.params.get('journal_guid') or '45b0969e-9b03-4f30-b4c6-b4b80ceff106'
```

Comment:
why do we have or condition here? seems like journal_guid is always going to be passed.